### PR TITLE
docs: Admin Partitions K8s requirement update

### PR DESCRIPTION
**Pull request by david-yu**
Clarify deployment model for Clients and Servers on K8s.

### CODE REVIEW

**Review by thisisnotashwin — APPROVED** [review]
Thanks for adding this clarification David!!

File: website/content/docs/enterprise/admin-partitions.mdx [hunk 1]
```diff
@@ -82,7 +82,8 @@ Your Consul configuration must meet the following requirements to use admin part
 
 One of the primary use cases for admin partitions is for enabling a service mesh across multiple Kubernetes clusters. The following requirements must be met to create admin partitions on Kubernetes:
 
-* Two or more Kubernetes clusters. Consul servers must be deployed to a single cluster. The other clusters should run Consul clients.
+* If deploying both Consul servers on Kubernetes, Consul servers must be deployed wihtin a single Kubernetes cluster. Consul servers may be deployed external to Kubernetes and configured using the `externalServers` stanza.
```
Comment (karl-cardenas-coding) [inline website/content/docs/enterprise/admin-partitions.mdx:85]:
```suggestion
* If deploying both Consul servers on Kubernetes, each Consul server must be deployed in its own Kubernetes cluster. Consul servers may be deployed external to Kubernetes and configured using the `externalServers` stanza. 
```
Thoughts on this? I tried to make it a bit more clearer.

**Review by karl-cardenas-coding — APPROVED** [review]
👍🏼

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/563785.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 6689425bec1839e69222dcfc9efb85512eb185f5 onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 6689425bec1839e69222dcfc9efb85512eb185f5 onto `release/1.11.x` succeeded!

**Pull request merged**